tests/xharness/Xharness.Tests/Xharness.Tests.csproj

```diff
@@ -159,6 +165,21 @@
     <Compile Include="BCLTestImporter\Tests\TestAssemblyDefinitionTest.cs" />
     <Compile Include="BCLTestImporter\Tests\TestProjectDefinitionTest.cs" />
     <Compile Include="Tests\XmlResultParserTests.cs" />
+    <Compile Include="..\Logging\ILogs.cs">

```

Comment:
Any reason why not to reference the tested project but instead link all the files? I opened the solution and there were all the source files in the test project as links

Reply:
Step by step. The main project is not a lib, but an exe. Once testing is done, I'll add it as a ref.